keps/sig-storage/2923-csi-migration-ceph-rbd/README.md

```diff
@@ -49,5 +49,7 @@ Major milestones in the life cycle of a KEP should be tracked in `Implementation
 Major milestones for Ceph RBD in-tree plugin CSI migration:
 
 - 1.23
-  - CephRBD CSI migration to Alpha
+  - Ceph RBD CSI migration to Alpha
+- 1.24
+  - Ceph RBD CSI migration to Beta
```

Comment:
Can you state if this is off by default or on by default?

Reply:
Done.. thanks @Jiawei0227